e2e/pod.go

```diff
@@ -542,3 +553,73 @@ func validateRWOPPodCreation(
 
 	return nil
 }
+
+// verifySeLinuxMountOption verifies the SeLinux context MountOption added to PV.Spec.MountOption
+// is successfully used by nodeplugin during mounting by checking for its presence in the
+// nodeplugin container logs.
+func verifySeLinuxMountOption(
+	f *framework.Framework,
+	pvcPath, appPath, daemonSetName, cn, ns string,
+) error {
+	mountOption := "context=\"system_u:object_r:container_file_t:s0:c0,c1\""
```

Comment:
so. minikube does not have SELinux, and the contents of the  `context=` would therefore not matter, as it is ignored. What happens when you run this test on a system where SELinux is enforced, will mounting (or `createApp()` below) fail?

Reply:
> so. minikube does not have SELinux, and the contents of the `context=` would therefore not matter, as it is ignored. What happens when you run this test on a system where SELinux is enforced, will mounting (or `createApp()` below) fail?

No, this label would be applied on mount point during mounting. 
Depending on the pod security context selinux options(matches the passed context or not), CRI relabelling may or may not change the already applied label in general case after mounting by csidriver.

But with the KEP, feature gates and RWOP, kublet will pass this context as mountOption automatically. 
So this e2e tries to mimic the same. 
https://kubernetes.io/blog/2023/04/18/kubernetes-1-27-efficient-selinux-relabeling-beta/#mounting-with-selinux-context

Reply:
There's one case where selinux relabelling is not done by CRI, when selinux type is "spc_t" (privileged container).

I created a cephfs pvc with selinux label mountOption and mounted it with such a pod.
```
root@csicephfs-demo-pod-3:/#  ls -liaZ /var/lib/www/html
total 0
1099511627785 drwxr-xr-x. 2 root root system_u:object_r:container_file_t:s0:c244,c505  0 Jun 15 06:55 .
     98598060 drwxr-xr-x. 3 root root system_u:object_r:container_file_t:s0:c193,c596 18 Jun 15 07:01 ..```

Reply:
My concern was that if mounting fails, running e2e on a different environment fails as well, which would not be intended. Thanks for checking!